packages/fileimport-service/ifc/parser.js

```diff
@@ -4,9 +4,9 @@ const { logger } = require('../observability/logging.js')
 const ServerAPI = require('./api.js')
 
 module.exports = class IFCParser {
-  constructor({ serverApi, logger }) {
+  constructor({ db, serverApi, logger }) {
```

Comment:
this function is not used at all.
I only saw a reference to it in the readme.
Is it safe to delete it?

Reply:
I removed the old parser.js file as it was unused and renamed the parser_v2